COMPARE.md

```diff
@@ -114,8 +114,8 @@ Ramda was a massive influence on Tinkoff Utils It borrowed the same functional p
 
 
 ## [Lodash](https://lodash.com/)
-Be careful when migrating from lodash to Utils:
-1. Check function signature, since Utils uses data as the last argument in contrast with lodash
+When migrating from lodash to Utils, keep in mind that –
+1. function signatures, since Utils uses data as the last argument in contrast with lodash
```

Comment:
Ahem

Reply:
> keep in mind that
> function signatures
keep in mind that function signatures what?

Reply:
fixed